File: Rectangle.java
public class Rectangle extends Figure {
    private double width, height;

    public Rectangle(String color, double width, double height) {
        super(color);
        this.width = width;
        this.height = height;
    }

    @Override
    public double perimetro() {
        return 2 * (width + height);
    }

    @Override
    public double area() {
        return width * height;
    }
}

File: Rectangle.py
class Rectangle(Figure):
    def __init__(self, color, width, height):
        super().__init__(color)
        self.width = width
        self.height = height

    def perimetro(self):
        return 2 * (self.width + self.height)

    def area(self):
        return self.width * self.height
